rs/ic_os/partition_tools/src/ext.rs

```diff
@@ -353,4 +371,100 @@ mod test {
             Some("/boot"),
         );
     }
+
+    #[test]
+    fn debugfs_input_test() {
+        assert_eq!(
+            ExtPartition::debugfs_input(
+                Path::new("path/to/input.sh"),
+                Path::new("/opt/ic/bin/output.sh")
+            ),
+            indoc::formatdoc!(
+                "mkdir /opt
+                               mkdir /opt/ic
+                               mkdir /opt/ic/bin
+                               cd /opt/ic/bin
+                               rm output.sh
+                               write path/to/input.sh output.sh
+                               "
```

Comment:
Is the formatting here correct?

Reply:
Good catch. I formatted the string then the autoformatter moved the first line but didn't change the contents of the string.